gradle-plugin/src/main/java/com/squareup/anvil/plugin/AnvilPlugin.kt

```diff
@@ -357,10 +368,11 @@ internal class Variant private constructor(
   val name: String,
   val project: Project,
   val compileTaskProvider: TaskProvider<KotlinCompile>,
-  val androidVariant: BaseVariant?,
+  @Suppress("TYPEALIAS_EXPANSION_DEPRECATION")
+  val androidVariant: BaseVariantDeprecated?,
   val androidSourceSets: List<AndroidSourceSet>?,
   val compilerPluginClasspathName: String,
-  val variantFilter: VariantFilter,
+  val variantFilter: VariantFilter
```

Comment:
Well FWIW, it's disabled in .editorconfig:

https://github.com/square/anvil/blob/7507b2735c0167c2095d422f010396b255983f49/.editorconfig#L494-L495

This *error* (😉) would have been allowed in CI because we're still using KtLint `0.41.0`, which had no concept of trailing commas.